<ui version="4.0" >
 <class>DiscoItemsWindowClass</class>
 <widget class="QMainWindow" name="DiscoItemsWindowClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DiscoItemsWindow</string>
  </property>
  <property name="iconSize" >
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle" >
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget" >
   <layout class="QVBoxLayout" >
    <property name="leftMargin" >
     <number>3</number>
    </property>
    <property name="topMargin" >
     <number>3</number>
    </property>
    <property name="rightMargin" >
     <number>3</number>
    </property>
    <property name="bottomMargin" >
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lblJid" >
        <property name="text" >
         <string>Jid:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbJid</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbJid" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblNode" >
        <property name="text" >
         <string>Node:</string>
        </property>
        <property name="buddy" >
         <cstring>cmbNode</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbNode" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QGroupBox" name="grbActions" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title" >
         <string>Actions</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="trwItems" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="contextMenuPolicy" >
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="textElideMode" >
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="indentation" >
         <number>12</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Jid</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Node</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="tlbToolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="movable" >
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
